python/pyarrow/tests/test_dataset.py

```diff
@@ -385,10 +385,12 @@ def test_partitioning():
 
 def test_expression_serialization():
     a = ds.scalar(1)
+    a_ = ds.scalar(pa.scalar(1))
```

Comment:
Uh, can you use another name? Why not `g`?

Reply:
Updated.